README.md

```diff
@@ -1,49 +1,75 @@
 # GovWifi Authentication API
 
-Authentication API for GovWifi
-
-## Overview
+This API is used by the frontend when a connection request is made to GovWifi.
+Its function is to check these requests against user details and the result determines how FreeRadius reacts to it e.g. allow / dissallow the connection.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
+
+<!-- vscode-markdown-toc -->
+* 1. [Overview](#Overview)
+	* 1.1. [Sinatra routes](#Sinatraroutes)
+	* 1.2. [Dependencies](#Dependencies)
+* 2. [Developing](#Developing)
+	* 2.1. [Running the tests](#Runningthetests)
+	* 2.2. [Using the linter](#Usingthelinter)
+	* 2.3. [Serving the app locally](#Servingtheapplocally)
+	* 2.4. [Deploying changes](#Deployingchanges)
+* 3. [Gotchas](#Gotchas)
+	* 3.1. [Extra API parameters](#ExtraAPIparameters)
+* 4. [Licence](#Licence)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+##  1. <a name='Overview'></a>Overview
 
 GovWifi exposes FreeRADIUS servers on public IP addresses, which are configured
 to communicate with this private API via its REST plugin.
 
-### Sinatra routes
+###  1.1. <a name='Sinatraroutes'></a>Sinatra routes
```

Comment:
I wouldn't expose this detail here. I can find this out easily by going to `app.rb` once I know it's a Sinatra app.

I might describe it at a slightly high level if I thought it was valuable:
> This API has one endpoint (at /authorize/user)

But I'd probably just cut it out completely.